docs/RFCS/postgres_syntax.md

```diff
@@ -0,0 +1,98 @@
+- Feature Name: Postgres Syntax Compatibility
+- Status: draft
+- Start Date: 2017-04-16
+- Authors: Matt Jibson
+- RFC PR:
+- Cockroach Issue:
+
+# Summary
+
+This proposal is to define the specific goals of the two syntax types
+(traditional and modern) so we can correctly decide how to triage parsing
+differences between CockroachDB and Postgres, Spanner, or other databases.
+
+# Motivation
+
+The differences in syntax types (`traditional` and `modern`) in CockroachDB
+are not documented and have no defined goals or reasons for existing or
+differing. The unwritten reasons are that traditional is the old Postgres
+style and modern is the new Spanner style. However, we have already
+made some choices to differ from Postgres ([integer division returning
+decimals](https://github.com/cockroachdb/cockroach/pull/3308), for example)
+because we thought the Postgres syntax could be improved. This style worked
```

Comment:
"However, we have already made some choices to differ from Postgres (*followed by an example which has nothing to do with syntax*) ..."
I think this sentence can be removed. We did not change the postgres syntax for those features supported by postgres (we did add syntax, but only for constructs that would be invalid in pg).

Your example highlights we have created semantic differences. These differences exist independent of syntax. See my comments below.